src/go/k8s/pkg/resources/statefulset_update.go

```diff
@@ -394,9 +396,12 @@ func (r *StatefulSetResource) podEviction(ctx context.Context, pod, artificialPo
 		}
 		r.pandaCluster.SetDecommissionBrokerID(id)
 
-		if err = r.handleDecommissionInProgress(ctx, log); err != nil {
+		if err = r.handleDecommission(ctx, log); err != nil {
 			return err
 		}
+
+		utils.DeletePodPVCs(ctx, r.Client, pod, log)
```

Comment:
Looking at the rest of the changes why there is no debug logs?

Reply:
Mostly because I wasn't looking for a bug here. :grinning:

Reply:
Should we be consistent with the verbosity and add logs?